src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -833,15 +832,12 @@ public java.net.URL getMasterUrl() {
     @Override
     public String consoleURL() throws IOException {
         try {
-            if (isOpenShift()) {
-                OpenShiftClient oclient = toOpenShiftClient();
-                if (oclient != null) {
-                    VersionInfo info = oclient.getOpenShiftV3Version();
-                    if (info == null) {
-                        ConfigMap configMap = client.configMaps().inNamespace(OCP4_CONFIG_NAMESPACE).withName(OCP4_CONSOLE_PUBLIC_CONFIG_MAP_NAME).get();
-                        if (configMap != null) {
-                            return configMap.getData().get(OCP4_CONSOLE_URL_KEY_NAME);
-                        }
+            if (openshiftClient != null) {
+                VersionInfo info = openshiftClient.getOpenShiftV3Version();
+                if (info == null) {
+                    ConfigMap configMap = client.configMaps().inNamespace(OCP4_CONFIG_NAMESPACE).withName(OCP4_CONSOLE_PUBLIC_CONFIG_MAP_NAME).get();
```

Comment:
just tested out that this code needs to use openshiftClient versus client. tested out that it actually results the same, and will improve readability.